scripts.js

```diff
@@ -0,0 +1,66 @@
+let myLibrary = [];
+
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+function checkBooks(library) {
+  return library.length === 0 ? 'There are no books' : '';
```

Comment:
- [ ] [OPTIONAL] You can write this as 
`return !library.length ? 'There are no books' : ''`

since 0 is a falsy value.